### PR TITLE
[CI] Run coding-standard check on Travis

### PR DESCRIPTION
**Pull request by Slamdunk**
You are bored to:

1. Comment every time "*Thanks but we don't accept CS fix PRs as I don't want to encourage people to do this everywhere.*"
1. "*I run it myself every now and then.*"
1. Read contributions on Pull Request that aren't well formatted, and so you waste brain time reading code written on standards that aren't yours

Aren't you?

References:
* https://github.com/composer/composer/pull/6801
* https://github.com/composer/composer/pull/6727
* https://github.com/composer/composer/pull/6431
* https://github.com/composer/composer/pull/5327
* https://github.com/composer/composer/pull/3939
* https://github.com/composer/composer/pull/2993

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -48,6 +49,7 @@ install:
     - composer install $flags
     # install dependencies using composer from source
     - bin/composer install $flags
+    - if [ "$CS_CHECK" = 1 ]; then wget https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.2.12/php-cs-fixer.phar -O php-cs-fixer; fi
```
Comment (Slamdunk) [inline .travis.yml]: I'm downloading the phar release because I think you don't want to alter the composer.* files.

**Comment by alcohol** [comment]
I don't think we want to mark PR's as failed though simply because they did not pass the CS check. It is probably better suited to be run in the `after_script` section.

**Comment by Slamdunk** [comment]
If the build doesn't fail, the contributor supposes the PR is ok and no further actions are required.
So you (the maintainer) have to manually check the build and tell the contributor to fix the coding standards.
Or you'll merge the PR un-fixed and need to fix it manually.

Why keep adding manual steps?

If the build fails, it is up to the contributor to get it green, and you (the maintainer) only have to read the contribution once it's good.

**Comment by alcohol** [comment]
We don't want to annoy people who actually made the effort to submit a PR, to go through the hassle of having to fix little nitpicking things such as CS. Keeping the barrier to contribute as low as possible basically.

**Pull request closed without merging**